src/java.base/share/classes/java/util/random/package-info.java

```diff
@@ -0,0 +1,469 @@
+ * and {@link RandomGenerator#doubles doubles()} to create streams of randomly
+ * chosen values. The methods
+ * {@link RandomGenerator#nextGaussian nextGaussian()} and
+ * {@link RandomGenerator#nextExponential nextExponential()} draw floating-point
+ * values from nonuniform distributions. The method
+ * {@link RandomGenerator#period period()} returns information about the period
+ * of the generator instance.
+ *
+ * <p> For a multi-threaded application, one can repeat the preceding steps to
+ * create additional {@linkplain RandomGenerator RandomGenerators}, but often it is preferable
+ * to use methods of the one single initially created generator to create others
+ * like it. (One reason is that some generator algorithms, if asked to create a
+ * new set of generators all at once, can make a special effort to ensure that
+ * the new generators are statistically independent.) If the initial generator
+ * implements the interface {@link StreamableGenerator}, then the method
+ * {@link RandomGenerator#rngs rngs()} can be used to create a stream of
+ * generators. If this is a parallel stream, then it is easy to get parallel
+ * execution by using the {@link Stream#map map()} method on the stream.
+ *
+ * <p> For a multi-threaded application that forks new threads dynamically,
+ * another approach is to use an initial generator that implements the interface
+ * {@link RandomGenerator.SplittableGenerator}, which is then considered to
+ * "belong" to the initial thread for its exclusive use; then whenever any
+ * thread needs to fork a new thread, it first uses the
+ * {@link RandomGenerator#split split()} method of its own generator to create a
+ * new generator, which is then passed to the newly created thread for exclusive
+ * use by that new thread.
+ *
+ *
+ * <h2>Choosing a Random Number Generator Algorithm</h2>
+ *
+ * If an application requires a random number generator algorithm that is
+ * cryptographically secure, then it should use an instance of the class
+ * {@link java.security.SecureRandom}.
+ *
+ * <p> For applications (such as physical simulation, machine learning, and
+ * games) that do not require a cryptographically secure algorithm, this package
+ * provides multiple implementations of interface {@link RandomGenerator} that
+ * provide trade-offs among speed, space, period, accidental correlation, and
+ * equidistribution properties.
+ *
+ * <p> For applications with no special requirements,
+ * "L64X128MixRandom" has a good balance among speed, space,
+ * and period, and is suitable for both single-threaded and multi-threaded
+ * applications when used properly (a separate instance for each thread).
+ *
+ * <p> If the application uses only a single thread, then
+ * "Xoroshiro128PlusPlus" is even smaller and faster, and
+ * certainly has a sufficiently long period.
+ *
+ * <p> For an application running in a 32-bit hardware environment and using
+ * only one thread or a small number of threads, may be a good choice.
```

Comment:
I think the name of the suitable algorithm is missing here.

Reply:
Yes - "L32X64MixRandom". Thank you.